repo/packages/R/riak-kv/0/config.json

```diff
@@ -0,0 +1,259 @@
+                    "description": "Mesos authentication secret file.",
+                    "default": ""
+                },
+                "instances": {
+                    "type": "number",
+                    "description": "Number of framework instances",
+                    "default": 1
+                },
+                "healthcheck-grace-period-seconds": {
+                    "type": "number",
+                    "description": "Memory requirements",
+                    "default": 300
+                },
+                "healthcheck-interval-seconds": {
+                    "type": "number",
+                    "description": "Memory requirements",
+                    "default": 60
+                },
+                "healthcheck-timeout-seconds": {
+                    "type": "number",
+                    "description": "Memory requirements",
+                    "default": 20
+                },
+                "healthcheck-max-consecutive-failures": {
+                    "type": "number",
+                    "description": "Memory requirements",
+                    "default": 5
+                },
+                "constraints": {
+                    "type": "array",
+                    "items": {
+                        "type": "array",
+                        "items": [
+                            {
+                                "type": "string"
+                            },
+                            {
+                                "type": "string",
+                                "enum": ["UNIQUE", "CLUSTER", "GROUP_BY", "LIKE", "UNLIKE"]
+                            },
+                            {
+                                "type": "string"
+                            }
+                        ],
+                        "minItems": 2,
+                        "additionalItems": false
+                    },
+                    "description": "Marathon constraints for the framework/scheduler instance. Valid constraint operators are one of UNIQUE, CLUSTER, GROUP_BY, LIKE, UNLIKE.",
+                    "default": []
+                },
+                "scheduler": {
+                    "type": "object",
+                    "description": "Requirements for Riak Mesos Scheduler",
+                    "properties": {
+                        "url": {
+                            "type": "string",
+                            "description": "Package location for Riak Mesos Scheduler tarball.",
+                            "default": "https://github.com/basho-labs/riak-mesos-scheduler/releases/download/1.8.1/riak_mesos_scheduler-1.8.1-mesos-1.0.0-ubuntu-14.04.tar.gz"
+                        },
+                        "package": {
+                            "type": "string",
+                            "description": "Package name for Riak Mesos Scheduler tarball.",
+                            "default": "riak_mesos_scheduler-1.8.1-mesos-1.0.0-ubuntu-14.04.tar.gz"
```

Comment:
We use these `package` config items to inform the Scheduler about the artifact names it will have, as we serve them ourselves for Executors to download within the cluster.

This specific instance `.riak.scheduler.package` might actually be superfluous.